pkg/aws/dynamo_locking.go

```diff
@@ -16,6 +16,32 @@ type DynamoDbLock struct {
 	DynamoDb *dynamodb.DynamoDB
 }
 
+func (dynamoDbLock *DynamoDbLock) waitUntilTableCreated() error {
+	input := &dynamodb.DescribeTableInput{
+		TableName: aws.String(TABLE_NAME),
+	}
+	status, err := dynamoDbLock.DynamoDb.DescribeTable(input)
+	cnt := 0
+
+	fmt.Printf("checking status, current status is %v", status.Table.TableStatus)
+	for err != nil && *(status.Table.TableStatus) != "ACTIVE" {
+		time.Sleep(1)
```

Comment:
this is nanoseconds

Reply:
good catch, there are so many docs pages where this is not documented correctly